trace-utils/src/span/v04/span.rs

```diff
@@ -49,13 +51,40 @@ impl FromStr for SpanKey {
     }
 }
 
+/// Trait representing the requirements for a type to be used as a Span "string" type.
+/// Note: Borrow<str> is not required by the derived traits, but allows to access HashMap elements
+/// from a static str.
+pub trait SpanValue: Eq + Hash + Borrow<str> {}
```

Comment:
`SpanValue` is too broad as name, given this is explicitly targeting the strings, I would prefer `SpanString` to narrow down the scope.